frontends/bsd/src/app_root.tsx

```diff
@@ -238,6 +240,14 @@ export function AppRoot({
     lastScannedSheetIdx,
   ]);
 
+  useEffect(() => {
+    if (currentUserSession?.authenticated && smartcard.status === 'no_card') {
+      setRemoveCardScreenShown(true);
```

Comment:
Interesting - this logic feels flipped to me, since it seems that we should show the card-removal prompt when you're authenticated and there _is_ a card plugged in (not when you're authenticated and there isn't a card plugged in)

I'm likely missing something!

Reply:
I think this is a result of poor variable naming. By `removeCardScreenShown` I meant "the remove card screen has already been shown," not "the remove card screen is shown." I will rename to something clearer.

Reply:
Ah paired with your comment below (https://github.com/votingworks/vxsuite/pull/1993#issuecomment-1162528287), I understand now!